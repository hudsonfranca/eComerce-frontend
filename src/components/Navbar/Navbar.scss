@import "../../styles/scss/variables";
@import "../../styles/scss/media";
@import "../../styles/scss/HamburguerButton";

@mixin ToobarButton {
  .toobar__button {
    width: 40px;
    height: 40px;
    background-color: transparent;
    color: $secondaryColor;
    transition: 0.4s ease-in-out;
    border-radius: 5px;
    cursor: pointer;

    &:hover,
    &:active {
      color: lighten($color: $secondaryColor, $amount: 10);
    }
  }
}

.App-header {
  display: flex;
  flex-direction: column;
  background-color: transparent;
  width: 100%;
  height: 120px;
  position: fixed;
  z-index: 11;

  @include for-tablet-portrait-up {
    height: 65px;
    padding: 0;
  }

  .toobar {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    width: 100%;
    height: 100%;
    background-color: $primaryColor;
    border-bottom: 1px solid rgba($color: #000000, $alpha: 0.1);
    padding: 5px;
    box-shadow: 0px 6px 16px 0px $secondaryColor;

    @include for-tablet-portrait-up {
      display: grid;
      grid-template-columns: 1fr 1fr 4fr 1fr 1fr 1fr;
      grid-template-rows: 1fr;
      align-items: center;
      justify-items: center;
    }

    @include for-tablet-landscape-up {
      grid-template-columns: 1fr 4fr 1fr 1fr 1fr;
      grid-template-rows: 1fr;
    }

    .logo {
      width: 60px;
      height: 50px;
      cursor: pointer;
      border-radius: 50%;
      // @include for-phone-only {
      //   width: px;
      //   height: 95%;
      // }
      .logo-image {
        width: 100%;
        height: 100%;
        border-radius: 50%;
      }
    }

    .search {
      display: flex;
      flex-direction: row;
      justify-content: center;
      align-items: center;
      width: 100%;
      height: 40%;
      position: absolute;
      top: 60px;
      background-color: $primaryColor;
      left: 0;

      @include for-tablet-portrait-up {
        position: static;
      }

      .searchInput {
        width: 83%;
        height: 40px;
        border-bottom-left-radius: 5px;
        border-top-left-radius: 5px;
        padding: 0;
        font-size: 1em;
        border: 4px solid $secondaryColor;
        border-right: none;
      }

      .searchInputButom {
        width: 50px;
        height: 40px;
        border-bottom-right-radius: 5px;
        border-top-right-radius: 5px;
        color: rgba($color: #000000, $alpha: 0.5);
        border-left: none;
        background-color: $secondaryColor;
        cursor: pointer;
        transition: 0.5s;
        &:hover {
          color: $primaryColor;
          background-color: lighten($color: $secondaryColor, $amount: 10);
        }
      }
    }

    label {
      display: none;
      cursor: pointer;
      @include for-tablet-landscape-up {
        display: inline;
        font-size: 1rem;
        color: rgba($color: #000000, $alpha: 0.5);
        @if $primaryColor == #313131 {
          color: $secondaryColor;
        }
      }
    }

    .cart {
      @include ToobarButton();
    }

    .account-mobile {
      @include ToobarButton();
      @include for-tablet-landscape-up {
        display: none;
      }
    }

    .account-desktop {
      display: none;
      @include ToobarButton();
      @include for-tablet-landscape-up {
        display: inline-block;
      }
    }

    .favorites {
      @include ToobarButton();
    }

    .hamburguer-button {
      @include for-tablet-landscape-up {
        display: none;
      }
    }
  }
}
