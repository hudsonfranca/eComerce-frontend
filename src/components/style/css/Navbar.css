.hamburguerButton {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  width: 40px;
  height: 40px;
  border-radius: 5px;
  background: #f9f6f7;
  cursor: pointer;
  -webkit-transition: 0.5s;
  transition: 0.5s;
}

.hamburguerButton:hover, .hamburguerButton:active {
  background-color: white;
}

.hamburguerButton .rbline {
  width: 30px;
  height: 3px;
  background-color: #313131;
  border-radius: 5px;
}

.hamburguerButton.close {
  -webkit-transform: rotate(180deg);
          transform: rotate(180deg);
}

.hamburguerButton.close .rbline:nth-child(1) {
  -webkit-transform: rotate(45deg) translate(8px, 8px);
          transform: rotate(45deg) translate(8px, 8px);
}

.hamburguerButton.close .rbline:nth-child(2) {
  opacity: 0;
}

.hamburguerButton.close .rbline:nth-child(3) {
  -webkit-transform: rotate(-45deg) translate(6px, -7px);
          transform: rotate(-45deg) translate(6px, -7px);
}

.App-header {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  background-color: transparent;
  width: 100%;
  height: 120px;
  position: fixed;
}

@media (min-width: 600px) {
  .App-header {
    height: 65px;
    padding: 0;
  }
}

.App-header .toobar {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
      -ms-flex-direction: row;
          flex-direction: row;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
  width: 100%;
  height: 100%;
  background-color: #313131;
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
  padding: 5px;
  -webkit-box-shadow: 0px 6px 16px 0px rgba(50, 50, 50, 0.73);
          box-shadow: 0px 6px 16px 0px rgba(50, 50, 50, 0.73);
}

@media (min-width: 600px) {
  .App-header .toobar {
    display: -ms-grid;
    display: grid;
    -ms-grid-columns: 1fr 1fr 4fr 1fr 1fr 1fr;
        grid-template-columns: 1fr 1fr 4fr 1fr 1fr 1fr;
    -ms-grid-rows: 1fr;
        grid-template-rows: 1fr;
    -webkit-box-align: center;
        -ms-flex-align: center;
            align-items: center;
    justify-items: center;
  }
}

@media (min-width: 900px) {
  .App-header .toobar {
    -ms-grid-columns: 1fr 4fr 1fr 1fr 1fr;
        grid-template-columns: 1fr 4fr 1fr 1fr 1fr;
    -ms-grid-rows: 1fr;
        grid-template-rows: 1fr;
  }
}

.App-header .toobar .logo {
  cursor: pointer;
  color: #f9f6f7;
}

.App-header .toobar .search {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
      -ms-flex-direction: row;
          flex-direction: row;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  width: 100%;
  height: 40%;
  position: absolute;
  top: 60px;
  background-color: #313131;
  left: 0;
}

@media (min-width: 600px) {
  .App-header .toobar .search {
    position: static;
  }
}

.App-header .toobar .search .searchInput {
  width: 83%;
  height: 40px;
  border-bottom-left-radius: 5px;
  border-top-left-radius: 5px;
  padding: 0;
  font-size: 1em;
  border: 4px solid #f9f6f7;
  border-right: none;
}

.App-header .toobar .search .searchInputButom {
  width: 50px;
  height: 40px;
  border-bottom-right-radius: 5px;
  border-top-right-radius: 5px;
  color: rgba(0, 0, 0, 0.5);
  border-left: none;
  background-color: #f9f6f7;
  cursor: pointer;
  -webkit-transition: 0.5s;
  transition: 0.5s;
}

.App-header .toobar .search .searchInputButom:hover {
  color: #313131;
  background-color: white;
}

.App-header .toobar label {
  display: none;
  cursor: pointer;
}

@media (min-width: 900px) {
  .App-header .toobar label {
    display: inline;
    font-size: 1rem;
    color: rgba(0, 0, 0, 0.5);
    color: #f9f6f7;
  }
}

.App-header .toobar .cart .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #f9f6f7;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .cart .toobar__button:hover, .App-header .toobar .cart .toobar__button:active {
  color: white;
}

.App-header .toobar .account-mobile .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #f9f6f7;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .account-mobile .toobar__button:hover, .App-header .toobar .account-mobile .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .account-mobile {
    display: none;
  }
}

.App-header .toobar .account-desktop {
  display: none;
}

.App-header .toobar .account-desktop .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #f9f6f7;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .account-desktop .toobar__button:hover, .App-header .toobar .account-desktop .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .account-desktop {
    display: inline-block;
  }
}

.App-header .toobar .favorites .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #f9f6f7;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .favorites .toobar__button:hover, .App-header .toobar .favorites .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .hamburguer-button {
    display: none;
  }
}
/*# sourceMappingURL=Navbar.css.map */