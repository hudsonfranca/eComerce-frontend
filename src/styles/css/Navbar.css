.hamburguerButton {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  width: 40px;
  height: 40px;
  border-radius: 5px;
  background: #fafbfc;
  cursor: pointer;
  -webkit-transition: 0.5s;
  transition: 0.5s;
}

.hamburguerButton:hover, .hamburguerButton:active {
  background-color: white;
}

.hamburguerButton .rbline {
  width: 30px;
  height: 3px;
  background-color: #313131;
  border-radius: 5px;
}

.hamburguerButton.close {
  -webkit-transform: rotate(180deg);
          transform: rotate(180deg);
}

.hamburguerButton.close .rbline:nth-child(1) {
  -webkit-transform: rotate(45deg) translate(8px, 8px);
          transform: rotate(45deg) translate(8px, 8px);
}

.hamburguerButton.close .rbline:nth-child(2) {
  opacity: 0;
}

.hamburguerButton.close .rbline:nth-child(3) {
  -webkit-transform: rotate(-45deg) translate(6px, -7px);
          transform: rotate(-45deg) translate(6px, -7px);
}

.App-header {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  background-color: transparent;
  width: 100%;
  height: 120px;
  position: fixed;
  top: 0;
  z-index: 11;
}

.App-header .toobar {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
      -ms-flex-direction: row;
          flex-direction: row;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
  width: 100%;
  height: 100%;
  background-color: #313131;
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
  padding: 5px;
  -webkit-box-shadow: 0px 6px 16px 0px #fafbfc;
          box-shadow: 0px 6px 16px 0px #fafbfc;
}

@media (min-width: 600px) {
  .App-header .toobar {
    display: -ms-grid;
    display: grid;
    -ms-grid-columns: 1fr 1fr 4fr 1fr 1fr 1fr;
        grid-template-columns: 1fr 1fr 4fr 1fr 1fr 1fr;
    -ms-grid-rows: 1fr;
        grid-template-rows: 1fr;
    -webkit-box-align: center;
        -ms-flex-align: center;
            align-items: center;
    justify-items: center;
  }
}

@media (min-width: 900px) {
  .App-header .toobar {
    -ms-grid-columns: 1fr 4fr 1fr 1fr 1fr;
        grid-template-columns: 1fr 4fr 1fr 1fr 1fr;
    -ms-grid-rows: 1fr;
        grid-template-rows: 1fr;
  }
}

.App-header .toobar .logo {
  width: 60px;
  height: 50px;
  cursor: pointer;
  border-radius: 50%;
}

.App-header .toobar .logo .logo-image {
  width: 100%;
  height: 100%;
  border-radius: 50%;
}

.App-header .toobar .search {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
      -ms-flex-direction: row;
          flex-direction: row;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  width: 100%;
  height: 40%;
  position: absolute;
  top: 60px;
  background-color: #313131;
  left: 0;
}

@media (min-width: 600px) {
  .App-header .toobar .search {
    position: static;
  }
}

.App-header .toobar .search .searchInput {
  width: 83%;
  height: 40px;
  border-bottom-left-radius: 5px;
  border-top-left-radius: 5px;
  padding: 0;
  font-size: 1em;
  border: 4px solid #fafbfc;
  border-right: none;
}

.App-header .toobar .search .searchInputButom {
  width: 50px;
  height: 40px;
  border-bottom-right-radius: 5px;
  border-top-right-radius: 5px;
  color: rgba(0, 0, 0, 0.5);
  border-left: none;
  background-color: #fafbfc;
  cursor: pointer;
  -webkit-transition: 0.5s;
  transition: 0.5s;
}

.App-header .toobar .search .searchInputButom:hover {
  color: #313131;
  background-color: white;
}

.App-header .toobar label {
  display: none;
  cursor: pointer;
}

@media (min-width: 900px) {
  .App-header .toobar label {
    display: inline;
    font-size: 1rem;
    color: rgba(0, 0, 0, 0.5);
    color: #fafbfc;
  }
}

.App-header .toobar .cart .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #fafbfc;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .cart .toobar__button:hover, .App-header .toobar .cart .toobar__button:active {
  color: white;
}

.App-header .toobar .account-mobile .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #fafbfc;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .account-mobile .toobar__button:hover, .App-header .toobar .account-mobile .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .account-mobile {
    display: none;
  }
}

.App-header .toobar .account-desktop {
  display: none;
}

.App-header .toobar .account-desktop .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #fafbfc;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .account-desktop .toobar__button:hover, .App-header .toobar .account-desktop .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .account-desktop {
    display: inline-block;
  }
}

.App-header .toobar .favorites .toobar__button {
  width: 40px;
  height: 40px;
  background-color: transparent;
  color: #fafbfc;
  -webkit-transition: 0.4s ease-in-out;
  transition: 0.4s ease-in-out;
  border-radius: 5px;
  cursor: pointer;
}

.App-header .toobar .favorites .toobar__button:hover, .App-header .toobar .favorites .toobar__button:active {
  color: white;
}

@media (min-width: 900px) {
  .App-header .toobar .hamburguer-button {
    display: none;
  }
}
/*# sourceMappingURL=Navbar.css.map */