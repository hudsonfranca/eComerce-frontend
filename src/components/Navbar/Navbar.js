import React from "react";
import { connect } from "react-redux";
import PropTypes from "prop-types";
import { mobileAccount, mobileCategories } from "../../actions/SideDrawer";

import "../style/css/Navbar.css";

export function Navbar({ mobileCategories, mobileAccount }) {
  return (
    <>
      <header className="App-header">
        <nav className="toobar">
          <div className="hamburguer-button">
            <button
              type="button"
              className="hamburguerButton"
              onClick={mobileCategories}
            >
              <div className="rbline" />
              <div className="rbline" />
              <div className="rbline" />
            </button>
          </div>
          <div className="logo">
            <h1>LOGO</h1>
          </div>
          <div className="search">
            <input type="text" className="searchInput" />
            <button type="button" className="searchInputButom">
              <i className="fas fa-search fa-2x" />
            </button>
          </div>
          <div className="account-mobile">
            <button
              type="button"
              className="toobar__button"
              id="account"
              onClick={mobileAccount}
            >
              <i className="fas fa-user-alt fa-2x " />
            </button>
            <label htmlFor="account" className="label">
              Sign in
            </label>
          </div>
          <div className="account-desktop">
            <button type="button" className="toobar__button" id="account">
              <i className="fas fa-user-alt fa-2x " />
            </button>
            <label htmlFor="account-desktop" className="label">
              Sign in
            </label>
          </div>
          <div className="favorites">
            <button type="button" className="toobar__button" id="favorites">
              <i className="fas fa-heart fa-2x " />
            </button>

            <label htmlFor="favorites" className="label">
              Favorites
            </label>
          </div>
          <div className="cart">
            <button type="button" className="toobar__button" id="cart">
              <i className="fas fa-shopping-cart fa-2x " />
            </button>

            <label htmlFor="cart" className="label">
              Cart
            </label>
          </div>
        </nav>
      </header>
    </>
  );
}

const mapStateToProps = state => ({});

const mapDispatchToProps = dispatch => ({
  mobileAccount: () => dispatch(mobileAccount()),
  mobileCategories: () => dispatch(mobileCategories())
});

export default connect(mapStateToProps, mapDispatchToProps)(Navbar);
