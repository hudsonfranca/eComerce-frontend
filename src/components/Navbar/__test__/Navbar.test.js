import React from "react";
import ReactDOM from "react-dom";
import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import { Navbar } from "../Navbar";

afterEach(cleanup);

it("Navbar renders without crashing", () => {
  const div = document.createElement("div");

  ReactDOM.render(<Navbar />, div);
});

it("Navbar matches snapshot", () => {
  const tree = renderer.create(<Navbar />).toJSON();

  expect(tree).toMatchSnapshot();
});
