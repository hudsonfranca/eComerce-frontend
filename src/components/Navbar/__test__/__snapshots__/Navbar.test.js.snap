// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches snapshot 1`] = `
<header
  className="App-header"
>
  <nav
    className="toobar"
  >
    <div
      className="hamburguer-button"
    >
      <button
        className="hamburguerButton"
        type="button"
      >
        <div
          className="rbline"
        />
        <div
          className="rbline"
        />
        <div
          className="rbline"
        />
      </button>
    </div>
    <div
      className="logo"
    >
      <h1>
        LOGO
      </h1>
    </div>
    <div
      className="search"
    >
      <input
        className="searchInput"
        onChange=""
        type="text"
      />
      <button
        className="searchInputButom"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-search fa-2x"
        />
      </button>
    </div>
    <div
      className="account-mobile"
    >
      <button
        className="toobar__button"
        id="account"
        type="button"
      >
        <i
          className="fas fa-user-alt fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="account"
      >
        Sign in
      </label>
    </div>
    <div
      className="account-desktop"
    >
      <button
        className="toobar__button"
        id="account"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-user-alt fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="account-desktop"
      >
        Sign in
      </label>
    </div>
    <div
      className="favorites"
    >
      <button
        className="toobar__button"
        id="favorites"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-heart fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="favorites"
      >
        Favorites
      </label>
    </div>
    <div
      className="cart"
    >
      <button
        className="toobar__button"
        id="cart"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-shopping-cart fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="cart"
      >
        Cart
      </label>
    </div>
  </nav>
</header>
`;

exports[`matches snapshot 1`] = `
<header
  className="App-header"
>
  <nav
    className="toobar"
  >
    <div
      className="hamburguer-button"
    >
      <button
        className="hamburguerButton"
        type="button"
      >
        <div
          className="rbline"
        />
        <div
          className="rbline"
        />
        <div
          className="rbline"
        />
      </button>
    </div>
    <div
      className="logo"
    >
      <h1>
        LOGO
      </h1>
    </div>
    <div
      className="search"
    >
      <input
        className="searchInput"
        onChange=""
        type="text"
      />
      <button
        className="searchInputButom"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-search fa-2x"
        />
      </button>
    </div>
    <div
      className="account-mobile"
    >
      <button
        className="toobar__button"
        id="account"
        type="button"
      >
        <i
          className="fas fa-user-alt fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="account"
      >
        Sign in
      </label>
    </div>
    <div
      className="account-desktop"
    >
      <button
        className="toobar__button"
        id="account"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-user-alt fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="account-desktop"
      >
        Sign in
      </label>
    </div>
    <div
      className="favorites"
    >
      <button
        className="toobar__button"
        id="favorites"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-heart fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="favorites"
      >
        Favorites
      </label>
    </div>
    <div
      className="cart"
    >
      <button
        className="toobar__button"
        id="cart"
        onClick=""
        type="button"
      >
        <i
          className="fas fa-shopping-cart fa-2x "
        />
      </button>
      <label
        className="label"
        htmlFor="cart"
      >
        Cart
      </label>
    </div>
  </nav>
</header>
`;
